class UserAlreadyRegisteredError(Exception):
    pass


class UserNotFoundError(Exception):
    pass
